### ISSUE TITLE
the value of automl isn't coming across

### ISSUE THREAD

**Issue by RachelKellam** [Azure-Samples/AzureMLWorkshop#27]
tracking this from a verbal feedback because is resonated with me

When just training a local script, its pretty straight forward (i.e., challenge 1A). But then when you go to automl which is supposed to make things easier... its longer, requires more input from me, and takes more time

The real value isn't shown because its just more work for me in the end

PM Team how can we help this?

**Comment by RachelKellam** [comment]
"can we get docs on all the overlapping concepts in automl"? 
We show pipeline kindof like child run, kindof like model. An automl pipeline produces a model which is equivalent of a child run?

Its all confusing

**Comment by RachelKellam** [comment]
"And when a user prints best run if we can see the best metric value to"

**Comment by CESARDELATORRE** [comment]
Agree. Let's follow up on this next week and discuss about it and how we can improve it, ok? 👍
